service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsAboutToSubmitControllerTest.java

```diff
@@ -85,14 +95,26 @@ void before() {
         given(documentSealingService.sealDocument(uploadedDocument)).willReturn(sealedDocument);
     }
 
-    @Test
-    void shouldCreateAdditionalApplicationsBundleWithC2DocumentWhenC2OrderIsSelectedAndSupplementsIncluded() {
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    void shouldCreateAdditionalApplicationsBundleWithC2DocumentWhenC2OrderIsSelectedAndSupplementsIncluded(
+        boolean servingOthersToggledOn) {
         Map<String, Object> data = new HashMap<>();
         data.put("additionalApplicationType", List.of("C2_ORDER"));
         data.putAll(createTemporaryC2Document());
         PBAPayment temporaryPbaPayment = createPbaPayment();
         data.put("temporaryPbaPayment", temporaryPbaPayment);
         data.put("applicantsList", createApplicantsDynamicList(APPLICANT));
+        data.put("others", Others.builder()
```

Comment:
I know that this is existing code but can we update it to use the case data class instead of a map